guides/common/modules/proc_importing-a-repository-from-a-web-server.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Importing_a_Repository_from_a_web_server_{context}"]
+= Importing a Repository from a web server
+
+You can import an exported repository directly from a web server into an organization on another {ProjectServer}.
+For more information about exporting the content of a repository, see xref:Exporting_a_Repository_{context}[].
+
+.Prerequisites
+* The exported files must be in a syncable format.
+* The exported files must be accessible via HTTP/HTTPS.
+* If the export contains any Red Hat repositories, the manifest of the importing organization must contain subscriptions for the products contained within the export.
+* The user importing the Content View version must have the _Content Importer_ Role.
+
+.Procedure
+. Select the organization into which you want to import.
+. To import the repository to {ProjectServer}, enter the following command:
++
+[subs="+quotes"]
+----
+# hammer content-import repository \
+--organization="_My_Organization_" \
```

Comment:
Above, you've used "organization-id". I think in most Hammer CLI example we use in fact "organization"; so :+1: here; maybe adjust the other instances.